fv/test-workload/test-workload.go

```diff
@@ -171,10 +171,19 @@ func main() {
 					return
 				}
 				err = utils.RunCommand("ip", "-6", "route", "add", "default", "via", hostIPv6Addr.String(), "dev", "eth0")
+				if err != nil {
+					log.WithField("hostIP", hostIPv6Addr.String()).WithError(err).Info("Failed to add IPv6 route to eth0.")
+				}
 
 				// Output the routing table to the log for diagnostic purposes.
-				utils.RunCommand("ip", "-6", "route")
-				utils.RunCommand("ip", "-6", "addr")
+				err = utils.RunCommand("ip", "-6", "route")
+				if err != nil {
```

Comment:
Sorry meant this one and the couple below as well.

Reply:
Fixed